pkgs/tools/typesetting/tex/texlive/default.nix

```diff
@@ -198,9 +211,22 @@ let
       operator = { pkg, ... }: pkgListToSets (pkg.tlDeps or []);
     });
 
+  assertions =
+    lib.assertMsg (tlpdbVersion.year == version.texliveYear) "TeX Live year in texlive does not match tlpdb.nix, refusing to evaluate" &&
+    lib.assertMsg (tlpdbVersion.frozen == version.final) "TeX Live final status in texlive does not match tlpdb.nix, refusing to evaluate";
+
 in
   tl // {
-    inherit bin combine tlpdb-nix;
+
+    tlpdb = {
+      # nested in an attribute set to prevent them from appearing in search
+      nix = tlpdbNix;
+      xz = tlpdbxz;
+      inherit version;
```

Comment:
I am undecided about this line – should we or should we not expose the `version` attribute set? The details are somewhat available already from `extraVersion`.

Reply:
Just as I ask the question, I realise it's better not to expose more than strictly necessary.